pilmoji/core.py

```diff
@@ -321,7 +321,7 @@ def text(
                     asset = asset.resize(size, Image.Resampling.LANCZOS)
 
                     ox, oy = emoji_position_offset
-                    self.image.paste(asset, (x + ox, y + oy), asset)
+                    self.image.paste(asset, (int(x + ox), int(y + oy)), asset)
```

Comment:
```suggestion
                    self.image.paste(asset, (round(x + ox), round(y + oy)), asset)
```

it's better than just `int` since you could want negative values